.github/workflows/e2e.gradle.workflow_dispatch.main.project-at-repo-root.slsa3.yml

```diff
@@ -0,0 +1,108 @@
+on:
+  schedule:
+    - cron: "0 15 * * *"
+  workflow_dispatch:
+    inputs:
+      trigger_build:
+        description: "internal: do not check"
+        required: false
+        default: false
+        type: boolean
+
+permissions: read-all
+
+concurrency: "e2e-gradle-workflow_dispatch-main-default-slsa3"
+
+env:
+  # TODO(#263): create dedicated token
+  GH_TOKEN: ${{ secrets.E2E_NODEJS_TOKEN }}
+  ISSUE_REPOSITORY: slsa-framework/slsa-github-generator
+
+jobs:
+  # Bootstrap
+  ################################################################################
+
+  bootstrap:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && !inputs.trigger_build)
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - env:
+          PACKAGE_DIR: ./
+        run: ./.github/workflows/scripts/e2e-gradle-push.sh
```

Comment:
e2e-grade-push creates a push event right? Maybe this test should be named `e2e.gradle.push.main.project-at-repo-root.slsa3.yml` ?

I see that the original one that you based this off of also has `workflow_dispatch` in the file name but it should also be `push` since it tests the `push` event.

Reply:
Ah, I see it does trigger `workflow_dispatch` after the push. Nevermind! :)